rtlib/include/rt/Data.hpp

```diff
@@ -0,0 +1,71 @@
+//
+// Created by Anthony Tamasi on 2019-10-17.
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include <boost/filesystem.hpp>
+#include <opencv2/core.hpp>
+
+#include "rt/ImageTypes.hpp"
+#include "rt/types/ITKMesh.hpp"
+
+namespace et = envitools;
+namespace fs = boost::filesystem;
+
+
+namespace Data {
```

Comment:
`namespace rt`

Reply:
Just so I'm clear, "namespace rt" should replace "namespace Data"?

Reply:
Instead of the namespace Data?